From b38cdebdfda910bd8f426ca9de33571422e072a7 Mon Sep 17 00:00:00 2001
From: Chien Nguyen <chien.nguyen.eb@rvc.renesas.com>
Date: Sat, 6 Aug 2016 04:20:03 +0700
Subject: [PATCH] Add FDPM into device tree of RZG1M

This patch file add device tree node of FDPM into RZG1M device tree

Signed-off-by: Chien Nguyen <chien.nguyen.eb@rvc.renesas.com>
---
 arch/arm/boot/dts/r8a7743.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7743.dtsi b/arch/arm/boot/dts/r8a7743.dtsi
index 941e288..bb30d61 100644
--- a/arch/arm/boot/dts/r8a7743.dtsi
+++ b/arch/arm/boot/dts/r8a7743.dtsi
@@ -946,6 +946,22 @@
 		renesas,#wpf = <4>;
 	};
 
+	fdp0: fdpm@fe940000 {
+		compatible = "renesas,fdp1";
+		reg = <0 0xfe940000 0 0x2400>;
+		interrupts = <0 262 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A7743_CLK_FDP1_0>;
+		power-domains = <&cpg_clocks>;
+	};
+
+	fdp1: fdpm@fe944000 {
+		compatible = "renesas,fdp1";
+		reg = <0 0xfe944000 0 0x2400>;
+		interrupts = <0 263 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A7743_CLK_FDP1_1>;
+		power-domains = <&cpg_clocks>;
+	};
+
 	du: display@feb00000 {
 		compatible = "renesas,du-r8a7743";
 		reg = <0 0xfeb00000 0 0x40000>,
-- 
1.9.1

